suggestCommands.pl

```diff
@@ -122,11 +122,15 @@
 
 print qq(
 
+# If your contigs are interspersed with long stretches of unknown nucleotides (NNN...NNN) you should split them:
+# needs: Python 3 and biopython
```

Comment:
I would add a `requirements.txt` file for the python script. 

We should change the README as well.